kyaml/openapi/openapi.go

```diff
@@ -50,6 +50,93 @@ type openapiData struct {
 	schemaInit bool
 }
 
+// precomputedIsNamespaceScoped precomputes IsNamespaceScoped for known types. This avoids Schema creation,
+// which is expensive
+var precomputedIsNamespaceScoped = map[yaml.TypeMeta]bool{
```

Comment:
how was this computed? did you have a script that parses the openapi document?

Reply:
Suggestion: if you had a script to do this, move `precomputedIsNamespaceScoped` to `kyaml/openapi/kubernetesapi/openapiinfo.go`, and add the contents of the script to to `kyaml/openapi/scripts/makeOpenApiInfoDotGo.sh` - that way, we can recompute this variable every time we update the openapi data.

Reply:
The test output shows the exact map in go syntax, so I just copy and pasted from the failure. If it changes, the test will output the diff that needs to be updated

Reply:
Ack, thanks for the explanation.

> The test output shows the exact map in go syntax, so I just copy and pasted from the failure. If it changes, the test will output the diff that needs to be updated

Can you please add this as a comment above the test, and also instructions for how to update the var in https://github.com/kubernetes-sigs/kustomize/tree/master/kyaml/openapi, perhaps a new subsection under `Update the built-in schema to a new version`?